types/node/index.d.ts

```diff
@@ -8281,6 +8299,7 @@ declare module "http2" {
         const HTTP_STATUS_CONTINUE: number;
         const HTTP_STATUS_SWITCHING_PROTOCOLS: number;
         const HTTP_STATUS_PROCESSING: number;
+        const HTTP_STATUS_EARLY_HINTS: number;
```

Comment:
Can you sort those?

Reply:
I think you can just sort the entire chunk.

Reply:
cool just did Object.keys(c).sort()